### PR TITLE
Mobile preview

### PR DESCRIPTION
**Pull request by bmallred**
**Note**: Please do not delete this branch yet since work is still being done.

To demonstrate the current status of the mobile updates we want to merge this in to `master` before it becomes stale.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <head>
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
```
Comment (toolness) [inline src/index.html]: What's the rationale for restricting zooming here?  [WCAG AA's Zoom/Magnification guidelines](https://www.w3.org/TR/mobile-accessibility-mapping/#zoom-magnification) recommend against this, and practical reasons for it are given in Adrian Roselli's [Don't Disable Zoom](http://adrianroselli.com/2015/10/dont-disable-zoom.html) post.

**Comment by toolness** [comment]
Oh heh, looks like it just got merged, so I will go ahead and file my a11y comments as separate issues.

File: src/index.html (hunk 1 above)
Comment (toolness) [inline src/index.html]: Filed this as #1392.

File: src/components/Navigation/NavigationToggle.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          <a href="javascript:;;" className="navigation-toggle" onClick={this.toggle}>
+            <i className="fa fa-times" aria-hidden="true"></i>
+            <span>Close navigation</span>
+          </a>
+        </div>
+      )
+    }
+
+    return (
+      <a href="javascript:;;" className="navigation-toggle mobile-visible" onClick={this.toggle}>
```
Comment (toolness) [inline src/components/Navigation/NavigationToggle.jsx]: Filed this as #1393!

File: src/App.jsx [hunk 3]
```diff
@@ -52,16 +52,19 @@ export class App extends React.Component {
   }
 
   render () {
-    let logoutButton = this.props.authenticated && this.props.twofactor
+    const logoutButton = this.props.authenticated && this.props.twofactor
         ? (<a href="#" onClick={this.logout} className="logout">{i18n.t('app.logout')}</a>)
         : null
+    const klassTitle = `eapp-structure-right eapp-title ${this.props.settings.mobileNavigation ? 'mobile-hidden' : 'visible'}`.trim()
+    const klassNavigation = `eapp-structure-left eapp-navigation ${this.props.settings.mobileNavigation ? 'mobile-visible' : 'mobile-hidden'}`.trim()
+    const klassCore = `eapp-structure-right eapp-core ${this.props.settings.mobileNavigation ? 'mobile-hidden' : 'visible'}`.trim()
```
Comment (bmallred) [inline src/App.jsx]: @floresj and I had actually looked at it at one point and is still a possible future refactoring.

File: src/components/Navigation/NavigationToggle.jsx (hunk 2 above)
Comment (bmallred) [inline src/components/Navigation/NavigationToggle.jsx]: Thanks!

File: src/index.html (hunk 1 above)
Comment (bmallred) [inline src/index.html]: We'll update this to be compliant

**Comment by bmallred** [comment]
@toolness meant to merge it last night. This epic isn't complete but people wanted to be able to preview it in the staging server after seeing it in the sprint review demonstration.

**Comment by toolness** [comment]
ah got it, thanks for the clarification!

Sent from my iPhone

On Jun 8, 2017, at 8:26 AM, Bryan Allred <notifications@github.com> wrote:

@toolness <https://github.com/toolness> meant to merge it last night. This
epic isn't complete but people wanted to be able to preview it in the
staging server after seeing it in the sprint review demonstration.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/truetandem/e-QIP-prototype/pull/1387#issuecomment-307087846>,
or mute the thread

**Pull request merged**